### ISSUE TITLE
Build new RPM for 3.0

### ISSUE THREAD

**Issue by devicenull** [ceph/ceph-iscsi#64]
The latest RPM I can find for this is apparently 2.7: https://download.ceph.com/ceph-iscsi/latest/rpm/el7/noarch/

Are RPMs still being built for that?  Is that a recommended way to install the iscsi gateway?

**Comment by mikechristie** [comment]
Get up to date builds here:

https://shaman.ceph.com/repos/ceph-iscsi/

Note that between the 2.X and 3.X versions we merged ceph-iscsi-cli and ceph-iscsi-config into one rpm ceph-iscsi. However, in some older download dirs ceph-iscsi-cli and ceph-iscsi-config were in a dir also called ceph-iscsi.

So when upgrading manually remove ceph-iscsi-cli/ceph-iscsi-config and install ceph-iscsi.

**Comment by hoerup** [comment]
@mikechristie  all the 3.x builds of ceph-iscsi are no longer available on shaman

**Comment by mikechristie** [comment]
There was a bug in the spec related code that Jason just fixed yesterday. I see it in there today:

https://1.chacra.ceph.com/r/ceph-iscsi/wip-systemd-rhel7/35f0532990356e00e79656a8cdae379885be8725/centos/7/flavors/default/noarch/ceph-iscsi-3.0-54.g35f0532.el7.noarch.rpm

**Comment by mikechristie** [comment]
Oh no, that link was just for branch with Jason's fix. I will update this PR when a link to the master branch is ready.

**Comment by hoerup** [comment]
When 3.1 gets released - will it be published to download.ceph.com ?

**Comment by mikechristie** [comment]
Yes, I just found out that was broken yesterday, and Jason showed me how to fix it.. I am fixing up the last ceph-iscsi ceph-ansible support issues in my PR, then I will fix the build stuff.

**Comment by dsavineau** [comment]
The latest build on chacra is broken (in fact since 3.0-70 I think) because there's no python-rtslib 2.1.fb68 release available on chacra.
The dependency has been updated by https://github.com/ceph/ceph-iscsi/commit/e761753ef854589cbb69fa692a5854f800f02d2b

```console
Error: Package: ceph-iscsi-3.0-72.gc5ebf29.el7.noarch (ceph-iscsi-noarch)
           Requires: python-rtslib >= 2.1.fb68
           Available: python-rtslib-2.1.fb63-13.el7.noarch (base)
               python-rtslib = 2.1.fb63-13.el7
           Installing: python-rtslib-2.1.fb67-10.g7713d1e.noarch (python-rtslib-noarch)
               python-rtslib = 2.1.fb67-10.g7713d1e
```

**Comment by dillaman** [comment]
https://2.chacra.ceph.com/r/python-rtslib/master/67eb1605c697b6307d8083b2962f5170db13d306/centos/7/flavors/default/noarch/

**Comment by dsavineau** [comment]
@dillaman thanks for the quick update !

**Comment by ufm** [comment]
rpm -i https://1.chacra.ceph.com/r/ceph-iscsi/master/363d8f33b7818fb486443656a73c713cb473f53f/centos/7/flavors/default/noarch/ceph-iscsi-3.2-13.g363d8f3.el7.noarch.rpm
error: Failed dependencies:
        python-configshell >= 1.1.fb25 is needed by ceph-iscsi-3.2-13.g363d8f3.el7.noarch
        tcmu-runner >= 1.4.0 is needed by ceph-iscsi-3.2-13.g363d8f3.el7.noarch

Help me, please. Where I can get these rpm for centos7 ?

**Comment by mikechristie** [comment]
tcmu-runner:

https://1.chacra.ceph.com/r/tcmu-runner/master/9c84f7a4348ac326ac269fbdda507953dba6ec2c/centos/7/flavors/default/

configshell:

It is in RHEL 7.7 and 8.0. I do not know the time frame for when Centos will release their repose for those versions.

**Comment by ufm** [comment]
Thanx for help!

> configshell:
> 
> It is in RHEL 7.7 and 8.0. I do not know the time frame for when Centos will release their repose for those versions.

Hmm. What version of the Linux distribution to choose to put the gate with the least amount of problems? Except RHEL.
